platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -236,6 +244,48 @@ public void testInstallProfile() throws Exception {
     assertThat(options, hasItem(FeaturesService.Option.NoAutoRefreshBundles));
   }
 
+  @Test
+  public void testInstallFeatureCallIsPrivileged() throws Exception {
+    final AtomicReference<StackTraceElement[]> stackTraceElements = new AtomicReference<>();
+
+    doAnswer(
+            AdditionalAnswers.answerVoid(
+                o -> stackTraceElements.set(Thread.currentThread().getStackTrace())))
+        .when(mockFeaturesService)
+        .installFeature(anyString(), any(EnumSet.class));
+
+    ApplicationServiceBean serviceBean =
+        new ApplicationServiceBean(
+            testAppService, testConfigAdminExt, mBeanServer, mockFeaturesService);
+    serviceBean.installFeature("profile-name");
+
+    assertThat(
+        Arrays.stream(stackTraceElements.get())
```

Comment:
❗️ The problem here is that you have no way to know if the code that starts unit testing and ends up calling your test method doesn't already have the AccessController in the stack. The best approach would be to verify up the stack until you hit your test method and doing this inside a test case is not necessarily a good idea. I think this would be better done in a JUnit rule which would have access to the test method. You could then record the stack at the beginning and compare it with the stack at the end to make sure the AccessController is in the stack. While only doing this if the test method is annotated to check access controller. That would be reusable.